assingment.playground/Contents.swift

```diff
@@ -2,5 +2,41 @@ import Foundation
 
 private let workplace = Workplace()
 
-workplace.setEmptySeat(count: <#Int#>)
-workplace.employeesInOffice(employeeCount: <#Int#>)
+workplace.setEmptySeat(count: 9)
+workplace.employeesInOffice(employeeCount: 6)
+
+public func isSeatAvailable(_ seatIndex: Int) -> Bool {
+    let seat = workplace.seats[seatIndex]
```

Comment:
**Fun fact:** This could have been written as one-liner
`return workplace.seats[seatIndex] == nil ? true : false`

It is convenience way to write `if-else` where blocks execute only one command.
Formula goes as follows `is condition true` ? `yes` : `no`.